roles/neutron/vars/main.yml

```diff
@@ -83,6 +83,17 @@ __neutron_helm_values:
           tunnel_id_ranges: 1:1000
         ml2_type_vlan:
           network_vlan_ranges: external:1:4094
+      taas:
+        taas:
+          enabled: true
+    taas_plugin:
+      service_providers:
+        service_provider: >-
+          {% if atmosphere_network_backend == 'ovn' %}
+          TAAS:TAAS:neutron_taas.services.taas.service_drivers.ovn.taas_ovn.TaasOvnDriver:default
+          {% else %}
+          TAAS:TAAS:neutron_taas.services.taas.service_drivers.taas_rpc.TaasRpcDriver:default
+          {% endif %}
```

Comment:
Instead of Jinja, please use the `TaasRpcServer` here by default and add the `taas` specific configuration for OVN below.